src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
 
   return (
-    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4">
+    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4 ">
       <div className="flex items-center">
```

Comment:
Font size too small, should be 22px but was 18px

<img width="1778" alt="image" src="https://github.com/user-attachments/assets/f4b07a11-3f27-447e-830b-bfb23656c30b">